app/services-manager.ts

```diff
@@ -617,8 +615,10 @@ export class ServicesManager extends Service {
             }
 
             if (result.emitter === 'STREAM') {
-              const subject = new Subject<any>();
-              this.windowSubscriptions[result.resourceId] = subject;
+              let subject = this.windowSubscriptions[result.resourceId];
```

Comment:
Could this be an `or` statement, and return `this.windowSubscriptions[result.resourceId]`?